### PR TITLE
Fixes HTMLView placeholder/loading state

### PR DESCRIPTION
**Pull request by burkeshartsis**
### What does this PR do, or why is it needed?
This fixes the placeholder/loading state for the `HTMLView` by using the correct prop name 🙃 It also updates `withPlaceholder` to pass in `lineNumber` so that we can have more control over a `Paragraph` placeholder's length.
### What design trade-offs/decisions were made?
N/A
### How do I test this PR?
Check the Paragraph stories and click on some content in the app.
### What automated tests did you write?
N/A
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-htmlview/src/HTMLView/index.js [hunk 1]
```diff
@@ -81,9 +81,9 @@ class HTMLView extends PureComponent {
 
   render() {
     return (
-      <ParagraphPlaceholder lineNumber={8} onReady={!this.props.isLoading}>
+      <Paragraph lineNumber={8} isLoading={this.props.isLoading}>
```
Comment (conrad-vanl) [inline packages/apollos-ui-htmlview/src/HTMLView/index.js]: Actually, shouldn't this prop be `this.props.isLoading && !this.state.parsed`? We don't want to show the loading placeholder if we actually have content to render

Comment (vinnyjth) [reply packages/apollos-ui-htmlview/src/HTMLView/index.js]: Good catch!

Comment (burkeshartsis) [reply packages/apollos-ui-htmlview/src/HTMLView/index.js]:
```suggestion
      <Paragraph lineNumber={8} isLoading={this.props.isLoading && !this.state.parsed}>
```
